frontend/packages/pipelines-plugin/src/types/taskRun.ts

```diff
@@ -51,3 +51,15 @@ export const TaskRunGroupVersionKind: K8sGroupVersionKind = {
   version: 'v1',
   kind: 'TaskRun',
 };
+
+export type PipelineRunStatusType = {
+  Completed?: number;
+  Failed?: number;
+  Skipped?: number;
+  Cancelled?: number;
+  Incomplete?: number;
+  PipelineNotStarted?: number;
+  Pending?: number;
+  Running?: number;
+  Succeeded?: number;
```

Comment:
This type is used when we split the message and create the object out of it. We are not sure whether message includes these values. Based on this object we are creating object of type TaskStatus. Here we are making values to 0 if PipelineRunStatusType don't have the values. 

If I don't make it optional, it will throw error.